### DISCUSSION TITLE
What error is this and can I correct it?

### DISCUSSION THREAD

**Discussion by Joemoonsoo** [fmarotta/kaobook#213] [Q&A]
![Error](https://user-images.githubusercontent.com/97705744/153733755-e701c540-f8d0-474a-bfd7-77ffecb14c76.png)
![Error](https://user-images.githubusercontent.com/97705744/153733764-758de8ce-f8be-44ea-bc76-6e33b98df0bb.png)

**Comment by AlexanderZeilmann** [comment]
This problem is not related to kaobook, but have a [look at this answer](https://tex.stackexchange.com/a/579759).
